Mac compile instructions seem broken

I got this when running make from source after installing CMake:

```
make
[ 50%] Building CXX object CMakeFiles/ipsnect.dir/main.o
clang: warning: argument unused during compilation: '-static-libgcc' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '-static-libstdc++' [-Wunused-command-line-argument]
/Users/Username/Downloads/ipsnect-master/main.cpp:409:1: warning: non-void function does not return a value [-Wreturn-type]
}
^
1 warning generated.
[100%] Linking CXX executable ipsnect
clang: error: unsupported option '-static-libgcc'
make[2]: *** [ipsnect] Error 1
make[1]: *** [CMakeFiles/ipsnect.dir/all] Error 2
make: *** [all] Error 2
```

I dug up a Mac binary I compiled in 2019 though, and it still works.

Might be worth issuing a release with a Mac binary possibly?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 2.0)
 project (ipsnect)
 
 set(CMAKE_BUILD_TYPE Debug)
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -static-libgcc -static-libstdc++")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -static-libstdc++")
 
 file(GLOB _SRC
     "*.cpp"

--- a/main.cpp
+++ b/main.cpp
@@ -5,7 +5,7 @@
 
 using namespace std;
 
-const char* VERSION = "ipsnect utility v1.01";
+const char* VERSION = "ipsnect utility v1.03";
 
 struct Hunk
 {

